pom.xml

```diff
@@ -4,7 +4,7 @@
 
     <groupId>io.dropwizard</groupId>
     <artifactId>dropwizard-primer</artifactId>
-    <version>1.3.7-6</version>
+    <version>1.4.0-1</version>
```

Comment:
This version needs to be in the same release no sequence of dropwizard. To keep it aligned with dropwizard versions. Change it to 1.3.7-7